server/routing/lobby_test.go

```diff
@@ -91,12 +98,40 @@ func TestLobbyAPI(t *testing.T) {
 	if !assert.Len(t, lobbies[lobbyID].PlayerReady, 2) {
 		t.Fatal("expected 2 players in with a ready status in game lobby")
 	}
-	assert.Equal(t, lobbies[lobbyID].PlayerReady["TestPlayer1"], true)
-	assert.Equal(t, lobbies[lobbyID].PlayerReady["TestPlayer2"], false)
+	assert.Equal(t, lobbies[lobbyID].PlayerReady[TestPlayer1], true)
+	assert.Equal(t, lobbies[lobbyID].PlayerReady[TestPlayer2], false)
 
 	// Create game from our lobby
-	_, err = put(port, LobbyRoute+"/"+lobbyID+"/start", nil)
-	if !assert.Nil(t, err) {
+	
+
+	path := fmt.Sprintf("%v/%v/%v", pubsubRoute, lobbyID, TestPlayer1)
+	t.Logf("Path: %v", path)
+
+	u := url.URL{Scheme: "ws", Host: fmt.Sprintf(`localhost:%v`, port), Path: path}
+
+	t.Logf("Connecting to: %v", u)
+	c, _, err := websocket.DefaultDialer.Dial(u.String(), nil)
+	if err != nil {
+		t.Fatalf("dial: %v", err)
+	}
+	defer c.Close()
+
+	go func() {
+
+		_, err = put(port, LobbyRoute+"/"+lobbyID+"/start", nil)
+		if !assert.Nil(t, err) {
+			t.Fatal(err)
+		}
+	}()
+
+	_, message, err := c.ReadMessage()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	updatedLobby := pogo.GameStartMsg{}
```

Comment:
you may want to make some assertions so you know you got the right `GameStartMsg` (things such as gameID, and players)